db/dbtypes/types.go

```diff
@@ -490,13 +498,25 @@ func ChoiceIndexFromStr(choice string) (VoteChoice, error) {
 // VotingDone is the height at which voting is considered complete or when the
 // status changes from "started" to either "failed" or "lockedin".
 type MileStone struct {
-	ID         int64            `json:"-"`
-	Status     AgendaStatusType `json:"status"`
-	VotingDone int64            `json:"votingdone"`
-	Activated  int64            `json:"activated"`
-	HardForked int64            `json:"hardforked"`
-	StartTime  time.Time        `json:"starttime"`
-	ExpireTime time.Time        `json:"expiretime"`
+	ID            int64            `json:"-"`
+	Status        AgendaStatusType `json:"status"`
+	VotingStarted int64            `json:"votingStarted"`
+	VotingDone    int64            `json:"votingdone"`
+	Activated     int64            `json:"activated"`
+	HardForked    int64            `json:"hardforked"`
+	StartTime     time.Time        `json:"starttime"`
+	ExpireTime    time.Time        `json:"expiretime"`
+}
+
+// AgendaSummary describes a short summary of a given agenda that includes
+// vote choices tally and deployment rule change intervals.
+type AgendaSummary struct {
```

Comment:
There's also a type named `AgendaSummary` coming in #1125, but it's in the `agendas` package, so okay for now.